app/Http/routes.php

```diff
@@ -21,28 +21,28 @@
     '/',
     function () use ($app) {
         /** @var \App\Models\Project $project */
-        $project = Project::find(1);
-        $conditions = $project->conditions;
-        foreach ($conditions as $condition) {
-            $pipeline = new Pipeline();
-            $stream = new Stream();
-            $stream->project()->associate($project);
-            $stream->pipeable()->associate($condition);
-            $stream->save();
-
-            $traveler  = new Traveler($stream);
-            $traveler->bag->give([
-                'event' => [
-                    'type' => 'push'
-                ]
-            ]);
-
-            $pipeline
-                ->send($traveler)
-                ->flow($stream);
-        }
-
-        return $app->welcome();
+        // $project = Project::find(1);
+        // $conditions = $project->conditions;
+        // foreach ($conditions as $condition) {
+        //     $pipeline = new Pipeline();
+        //     $stream = new Stream();
+        //     $stream->project()->associate($project);
+        //     $stream->pipeable()->associate($condition);
+        //     $stream->save();
+        //
+        //     $traveler  = new Traveler($stream);
+        //     $traveler->bag->give([
+        //         'event' => [
+        //             'type' => 'push'
+        //         ]
+        //     ]);
+        //
+        //     $pipeline
+        //         ->send($traveler)
+        //         ->flow($stream);
+        // }
+        $message = str_replace('Lumen.', 'Pipes.', $app->welcome());
```

Comment:
Instead of commenting out could you move it to a debug route and check that the application is in debug mode.